docker/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -l
+
+openapi2confluence -p $1 -id $2 -s $3 -a $4 -t $5 -lid $6 -mid $7 -env $8 >> $GITHUB_OUTPUT
```

Comment:
Notei que estamos enviando o output do utilitário para o output da action, entretanto não definimos um output no arquivo `action.yml`.

Para definir um output em uma action precisamos seguir essa sintaxe: `{name}={value}` (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter).

Um output de uma action não é exibido por default em um workflow que utilizada o mesmo, assim talvez seja mais interessante enviar o output para `$GITHUB_STEP_SUMMARY` (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary)